source/dub/dub.d

```diff
@@ -350,6 +411,10 @@ class Dub {
 	@property const(string[string]) defaultPreRunEnvironments() const { return this.m_config.defaultPreRunEnvironments; }
 	@property const(string[string]) defaultPostRunEnvironments() const { return this.m_config.defaultPostRunEnvironments; }
 
+	private @property const(string) dubHome() const { return this.m_config.dubHome; }
+	private @property const(string) userSettingsOverride() const { return this.m_config.userSettings; }
```

Comment:
Why the `override` ? Also wouldn't we be better simply exposing `SpecialDir` ?
But again, why expose it ? I don't see any user, and YAGNI.

Reply:
Oh damn this is returning the config, not even the actual values! A big no from me.